README.rst

```diff
@@ -1,43 +1,107 @@
+dateparser -- python parser for human readable dates
+====================================================
 
-.. image:: https://badge.fury.io/py/dateparser.png
-    :target: http://badge.fury.io/py/dateparser
+.. image:: https://img.shields.io/travis/scrapinghub/dateparser/master.svg?style=flat-square
+    :target: https://travis-ci.org/scrapinghub/dateparser
+    :alt: travis build status
 
-.. image:: https://travis-ci.org/scrapinghub/dateparser.png?branch=master
-        :target: https://travis-ci.org/scrapinghub/dateparser
+.. image:: https://img.shields.io/pypi/dd/dateparser.svg?style=flat-square
+    :target: https://pypi.python.org/pypi/dateparser/
+    :alt: pypi downloads per day
 
-.. image:: https://pypip.in/d/dateparser/badge.png
-        :target: https://pypi.python.org/pypi/dateparser
+.. image:: https://img.shields.io/pypi/v/dateparser.svg?style=flat-square
+    :target: https://pypi.python.org/pypi/dateparser
+    :alt: pypi version
 
 
-Date parsing library designed to make it easy parsing dates commonly found in web pages
+`dateparser` provides modules to easily parse localized dates in almost
+any string formats commonly found on web pages.
 
 
-* Free software: BSD license
-* Documentation: https://dateparser.readthedocs.org.
-
 Features
 --------
+* Generic parsing of dates in English, Spanish, Dutch, Russian and several other langauges and formats.
+* Generic parsing of relative dates like: ``'1 min ago'``, ``'2 weeks ago'``, ``'3 months, 1 week and 1 day ago'``.
+* Generic parsing of dates with time zones abbreviations like: ``'August 14, 2015 EST'``, ``'July 4, 2013 PST'``.
+* Extensive test coverage.
+
+
+Usage
+-----
+The most straightforward way is to use the :func:`dateparser.parse` function,
+that wraps around most of the functionality in the module.
+
+.. automodule:: dateparser
+   :members: parse
+
+
+Popular Formats
++++++++++++++++
+
+    >>> import dateparser
+    >>> dateparser.parse('12/12/12')
+    datetime.datetime(2012, 12, 12, 0, 0)
+    >>> dateparser.parse(u'Fri, 12 Dec 2014 10:55:50')
+    datetime.datetime(2014, 12, 12, 10, 55, 50)
+    >>> dateparser.parse(u'Martes 21 de Octubre de 2014')  # Spanish
+    datetime.datetime(2014, 10, 21, 0, 0)
+    >>> dateparser.parse(u'Le 11 Décembre 2014 à 09:00')  # French
+    datetime.datetime(2014, 12, 11, 9, 0)
+    >>> dateparser.parse(u'13 января 2015 г. в 13:34')  # Russian
+    datetime.datetime(2015, 1, 13, 13, 34)
+    >>> dateparser.parse(u'1 เดือนตุลาคม 2005, 1:00 AM')  # Thai
+    datetime.datetime(2005, 10, 1, 1, 0)
+
+This will try to parse a date from the given string, attempting to
+detect the language each time.
 
-If you have needed to parse dates before, you probably have used the date
-parser of the dateutil_ module.
-We built this library on top of it, adding a few features:
+You can specify the language(s), if known, using ``languages`` argument. In this case, given languages, if supported, are used and language detection is skipped::
```

Comment:
We raise an error if language is unknown, don't we? Do we need "if supported part" here?